sessions/react-state-3/adding-animals/src/components/Form/index.js

```diff
@@ -0,0 +1,139 @@
+import "./Form.css";
+
+export default function Form({ onAddAnimal }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    onAddAnimal?.(data);
```

Comment:
```suggestion
     if(onAddAnimal) {
      onAddAnimal(data); 
    }		
```

I'm not sure if the optional chaining could be confusing. Codepen displays an eslint error:

> Expected an assignment or function call and instead saw an expression. (no-unused-expressions)eslint


Or we skip it because it's gives as prop from `App` component

```suggestion
    onAddAnimal(data);
```